src/uct/ib/configure.m4

```diff
@@ -226,10 +226,7 @@ AS_IF([test "x$with_ib" = "xyes"],
                       [[#include <infiniband/verbs.h>]])
 
        AS_IF([test "x$with_odp" = "xyes" ], [
-           AC_DEFINE([HAVE_ODP], 1, [ODP support])
-
-           AC_CHECK_DECLS(IBV_ODP_SUPPORT_IMPLICIT, [with_odp_i=yes], [],
-                          [[#include <infiniband/verbs.h>]])])
+           AC_DEFINE([HAVE_ODP], 1, [ODP support])])
```

Comment:
I would remove "with_odp" option altogether. seems it's not useful for anything anymore. WDYT?